css/index.css

```
@@ -807,91 +829,88 @@ body {
 /* Estilo para escritorio */
 @media (min-width: 768px) {
   .functionality {
-    height: 210px;
+      height: 210px;
   }
 
   .functionality__information {
-    flex-direction: row;
-    justify-content: space-between;
+      flex-direction: row;
+      justify-content: space-between;
   }
 
   .functionality__icons {
-    flex-direction: row;
-    align-items: center;
-    min-width: auto;
-    padding: 0;
-    gap: 13.32px;
+      flex-direction: row;
+      align-items: center;
+      min-width: auto;
+      padding: 0;
+      gap: 13.32px;
   }
 
   .functionality__text {
-    align-items: flex-start;
+      align-items: flex-start;
   }
 
   .functionality__paragraph, .functionality__Saber-mas {
-    text-align: left;
+      text-align: left;
   }
 
   .functionality__bord {
-    border: #F5F5F5 solid 1px;
-    height: 105px;
+      border: #F5F5F5 solid 1px;
+      height: 105px;
   }
 
   .carousel__controls, .carousel__indicators {
-    display: none;
+      display: none;
   }
 
   .functionality__carousel {
-    gap: 48px;
+      gap: 48px;
   }
 }
 
 
-/* Mobile-first left*/
 .most-popular {
-  width: 100%;
-  margin: auto;
-  padding: 20px; 
+  width: var(--width);
+  margin: var(--margin-auto);
+  padding: 20px
```

Creo que tienes una variable con este valor. Quizas tengas otros valores que ya tengas en variables y por ponerles un nombre muy atado a su seccion no las estes usando o incluso hasta creando varias variables con el mismo valor.
```suggestion
  padding: var(--margin-right-small);
```